src/ophyd_async/epics/adpilatus/_pilatus_io.py

```diff
@@ -19,5 +19,5 @@ def __init__(self, prefix: str, name: str = "") -> None:
         self.trigger_mode = epics_signal_rw_rbv(
             PilatusTriggerMode, prefix + "TriggerMode"
         )
-        self.armed_for_triggers = epics_signal_r(bool, prefix + "Armed")
+        self.armed_for_triggers = epics_signal_r(bool, prefix + "ArmedForTriggers")
```

Comment:
The PV is set in stone, we can only change the attribute name to match...
```suggestion
        self.armed = epics_signal_r(bool, prefix + "Armed")
```